src/frontend/src/components/Searching.tsx

```diff
@@ -0,0 +1,130 @@
+import { TextField, ToggleButton, ToggleButtonGroup } from "@mui/material";
```

Comment:
does search implements recommendation i.e you type a query and matching apartments are displayed directly?

Reply:
What do you mean? After you press enter, the results will appear.

Reply:
Make sure to fix your PR's linting since I removed the my-app directory

Reply:
> Make sure to fix your PR's linting since I removed the my-app directory

Actually the ESlint errors are due to some tree problem which I have not figured out yet.

Reply:
Try running "npm ls eslint" in the terminal. I get a ton of conflict warnings but I'm not sure how to resolve it

Reply:
Was it also failing before or only in this PR?